package.json

```diff
@@ -19,20 +18,21 @@
   ],
   "exports": {
     ".": {
-      "import": "./lib/esm/index.js",
-      "require": "./lib/commonjs/index.js"
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
     },
     "./bundle.css": "./lib/bundle.css",
     "./bundle-no-resets.css": "./lib/bundle-no-resets.css"
   },
   "files": [
     "lib",
     "src",
-    "THIRD-PARTY-NOTICES"
+    "THIRD-PARTY-NOTICES",
+    "dist"
   ],
   "scripts": {
-    "build": "rm -rf lib/** && npm run build:js && npm run build:css && npm run api-extractor && npm run generate-docs && npm run generate-notices",
-    "build:js": "tsc -p tsconfig.esm.json && tsc -p tsconfig.cjs.json",
+    "build": "rm -rf lib/** && rm -rf dist/** && npm run build:js && npm run build:css && npm run api-extractor && npm run generate-docs && npm run generate-notices",
+    "build:js": "tsc -p tsconfig.esm.json && tsc -p tsconfig.cjs.json && tsup src/index.ts --dts --sourcemap --format esm,cjs",
```

Comment:
I don't think you need both tsc and tsup

Reply:
What do you mean here? People import your package like `@yext/search-ui-react/lib/...`?

You're outputting to /dist now instead of /lib.

Reply:
I suppose we don't advertise it for this repo specifically, but for but search-core we do advertise the different import paths: https://github.com/yext/search-core#explanation-of-builds

It's sometimes useful to be able to import a specific module type rather than relying on build tools to pick it automatically

Reply:
You don't have to import the full file path however